package/post-install.sh

```diff
@@ -26,28 +26,49 @@ ddtrace.request_init_hook=${EXTENSION_AUTO_INSTRUMENTATION_FILE}
 EOF
 }
 
-function install_conf_d_file() {
+function install_conf_d_files() {
     INI_FILE_PATH="${EXTENSION_CFG_DIR}/$INI_FILE_NAME"
 
-    println "Creating ddtrace.ini"
-    println "\n"
+    if [[ ! -f $INI_FILE_PATH ]]; then
```

Comment:
Would it make sense to extract this to a function like "generate_config_file()"?